validator/src/main/java/com/amazon/aoc/validators/ContainerInsightMetricsValidator.java

```diff
@@ -0,0 +1,76 @@
+package com.amazon.aoc.validators;
```

Comment:
i guess we forgot to put license header in validator sometimes :)

Reply:
Thanks for the reminder!